config/app.js

```diff
@@ -3,6 +3,10 @@ const { joinPath } = require('../lib/utils')
 module.exports = {
   title: 'ZWave To MQTT',
   storeDir: process.env.STORE_DIR || joinPath(true, 'store'),
+  defaultUser: 'admin',
```

Comment:
do we really want default creds, can't auth be off by default?

Reply:
some folk might just run it behind a reverse proxy and use the restful http endpoint, so auth in this just presents a barrier + complexity

Reply:
> do we really want default creds, can't auth be off by default?

This is another point I need to implement, btw when it is on how do I allow creating users? I would like to prevent using sign up forms...

Reply:
your password setter could also allow changing the username?

Reply:
nope, at least not yet.

Reply:
well, I know it doesn't yet, it was more a suggestion to avoid signup forms

Reply:
I'm not sure I will add it, at least until someone shows me a relevant use case